src/streamlink/plugins/nrk.py

```diff
@@ -13,7 +13,7 @@
 
 _id_re = re.compile(r"/(?:program|direkte|serie/[^/]+)/([^/]+)")
 _url_re = re.compile(r"https?://(tv|radio).nrk.no/")
-_api_baseurl_re = re.compile(r'apiBaseUrl:\s*"(?P<baseurl>[^"]+)"')
+_api_baseurl_re = re.compile(r'''apiBaseUrl:\s*["'](?P<baseurl>[^"']+)["']''')
```

Comment:
:) What would be best is to have a JS object parser to handle those, like `dirtyjson` but that hasn't been updated for a while.